package.json

```diff
@@ -21,6 +21,8 @@
     "@js-joda/core": "3.2.0",
     "@js-joda/timezone": "2.5.0",
     "@types/express": "^4.17.13",
+    "@types/fusebit__add-on-sdk": "^3.1.0",
```

Comment:
Here and not under `devDependencies` for the same reason that I added `@types/express` and `@types/node-fetch` to this section.